src/lib-test/fuzzer.h

```diff
@@ -0,0 +1,16 @@
+#ifndef FUZZER_H
+#define FUZZER_H
+
+#define FUZZ_DATA(args) \
+	int LLVMFuzzerTestOneInput(args); \
+	int LLVMFuzzerTestOneInput(args) {
+
+#define FUZZ_STR(str_arg) \
+	int LLVMFuzzerTestOneInput(const uint8_t *_param_data, size_t _param_size); \
+	int LLVMFuzzerTestOneInput(const uint8_t *_param_data, size_t _param_size) { \
+		T_BEGIN { str_arg = t_strndup(_param_data, _param_size);
```

Comment:
should this be ((str_arg)) ?

Reply:
Again no, that would fail to compile then when using `FUZZ_STR(const char *str)`

Reply:
ah, it works like that.